js/interface.js

```diff
@@ -54,18 +54,18 @@ function fetchCurrentDataSourceEntries() {
     return source.find({});
   }).then(function (rows) {
     if (!rows || !rows.length) {
-      rows = [{id: 1, name: 'Sample row 1'}, {id: 2, name: 'Sample row 2'}];
+      rows = [{data: { id: 1, name: 'Sample row 1'}}, {data: {id: 2, name: 'Sample row 2'}}];
     }
 
     var $entries = $contents.find('#entries');
 
-    var tableHead = '<tr>' + Object.keys(rows[0]).map(function (column) {
+    var tableHead = '<tr>' + Object.keys(rows[0].data).map(function (column) {
       return '<td>' + column + '</td>';
     }).join('') + '</tr>';
 
     var tableBody = rows.map(function (row) {
-      return '<tr>' + Object.keys(row).map(function (key) {
-        return '<td>' + row[key] + '</td>';
+      return '<tr>' + Object.keys(row.data).map(function (key) {
```

Comment:
Because getting data from a datasource return the da source entry raw, not only the data.
This is good as we might need updatedAt/createdAt in a very near future.